specification/semantic_conventions.md

```diff
@@ -27,3 +28,34 @@ All Splunk distributions of OpenTelemetry,
 | Attribute  | Type | Description  | Examples  | Required |
 |---|---|---|---|---|
 | `splunk.distro.version` | string | The version number of the Splunk distribution being used. | `1.5.0` | Yes |
+
+Note: this section does not apply to Real User Monitoring libraries, as they do not use the OpenTelemetry Resource.
+
+## Real User Monitoring Spans and Attributes
+
+Real User Monitoring (RUM) libraries MUST set the `service.name` resource attribute to the value of the `applicationName`
+configuration property. This is the only resource attribute that RUM libraries are supposed to set because it's the only one
+the Zipkin exporter can understand.
+
+The following attributes MUST be added to all spans emitted by RUM libraries:
+
+| Name                  | Type   | Description                                                                                                       |
+| ----                  | ----   | -----------                                                                                                       |
+| `app`                 | string | The value of the `applicationName` property configured in `SplunkRum`; same as `service.name` resource attribute. |
+| `splunk.rumSessionId` | string | The RUM session ID                                                                                                |
+| `splunk.rum.version`  | string | Version of the RUM library                                                                                        |
+| `component`           | string | Name of the instrumentation that produced this span                                                               |
+
+The following attributes MAY be added to all spans emitted by RUM libraries:
+
+| Name                     | Type   | Description                                                                     |
+| ----                     | ----   | -----------                                                                     |
+| `deployment.environment` | string | Value of the `deploymentEnvironment` property configured in `SplunkRum`, if any |
+
+Mobile RUM libraries (iOS, Android) MUST add device and system information to all spans:
```

Comment:
We should have a must for browser info too!

Reply:
I've added `location.href` and `splunk.script_instance` (changed from `scriptInstance` to conform to OTel naming strategy).
@jtmalinowski please take a look at it, is there anything else we want to add here?

Reply:
These 2 are all we want to add for now.